### ISSUE TITLE
Main content area in wizard is identified as clickable

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4660]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.3.1 Info and relationship

**Test Environment:**
- Test URL: https://clarity.design/documentation/wizard
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto any one of the wizard buttons and press it.
3. Move focus to the main content area and observe screen reader announcement.

**Expected result:**
Screen reader should announce the content area appropriately with proper semantics.

**Actual result:**
Screen reader announces the main content area as clickable. 

Chris (@chlane)  mentioned that, it is intentionally done  to allow keyboard users to reach the scrollable area. 
Query: Need to check if it can be implemented conditionally, so that the area receives tab focus only when it is scrollable?

**Comment by chlane** [comment]
This is working as designed per Clarity. Clarity dialogs contain scrollable divs at all times (http://whatsock.com/tsg/Coding%20Arena/Scrollable%20Divs/Scrollable%20Div%20(Internal%20Content)/demo.htm) to allow sighted keyboard-only user to scroll overflow. NVDA will announce these as clickable and at times that there is no overflow there will be an inert tab stop.

The A11y Team recommends that this be made conditional somehow. Maybe the consumer could pass in an argument?
